src/dso_api/dynamic_api/fields.py

```diff
@@ -32,11 +32,17 @@ def get_url(self, obj, view_name, request, format=None):
         if hasattr(obj, "pk") and obj.pk in (None, ""):
             # Unsaved objects will not yet have a valid URL.
             return None
+        elif isinstance(obj, str):
+            # This is a loose relation
```

Comment:
Is dat zo zeker te zeggen? waar komt die str vandaan? Klinkt best fragiel

Reply:
klopt idd. Ik zal het iets safer maken